restapi/src/main/java/io/stargate/web/docsapi/service/DocumentService.java

```diff
@@ -124,7 +116,8 @@ private boolean isEmptyArray(Object v) {
           .configBuilder()
           .bind(
               "$..*",
-              (v, parsingContext) -> {
+              (v0, parsingContext) -> {
+                final JsonNode v = (JsonNode) v0;
```

Comment:
Is this cast here safe? Can it be anything else?

Reply:
As far as I understand it is safe and the only reason arg comes as `Object` is so that JsonSurfer API is identical across providers -- not sure why it does not use generics instead, but this way json-simple/gson/jackson json value types are opaque. I am guessing some JSON libraries might not have a common base type above `java.lang.Object` (I think org.json and json-lib might be like this).

Jackson specifically always produces `JsonNode` and no other values are possible. I am pretty sure similarly Gson only produced `JsonElement`s.